src/DonorProfiles/resources/js/app/components/donation-table/donation-row/index.js

```diff
@@ -14,18 +15,27 @@ const DonationRow = ( { donation } ) => {
 				{ form.title }
 			</div>
 			<div className="give-donor-profile-table__column">
-				{ payment.date }
+				<div className="give-donor-profile-table__donation-date">{ payment.date }</div>
+				<div className="give-donor-profile-table__donation-time">{ payment.time }</div>
 			</div>
 			<div className="give-donor-profile-table__column">
 				<div className="give-donor-profile-table__donation-status">
-					{ payment.status }
+					<div className="give-donor-profile-table__donation-status-indicator" style={ { background: payment.status.color } } />
+					<div className="give-donor-profile-table__donation-status-label">
+						{ payment.status.label }
+					</div>
 				</div>
+				{ payment.mode !== 'live' && (
+					<div className="give-donor-profile-table__donation-test-tag">
+						{ __( 'Test Donation', 'givewp' ) }
```

Comment:
@henryholtgeerts text-domain is `give`. Can you fix the typo?

Reply:
Done, ref: https://github.com/impress-org/givewp/pull/5566/commits/1e5d1a5ce58ef7b9d3ade6fe4766116b3cd85492